packages/suite/src/support/tests/configureStore.tsx

```
@@ -1,16 +1,26 @@
 import reduxMockStore, { MockStoreCreator } from 'redux-mock-store';
-import thunk from 'redux-thunk';
+import { withExtraArgument } from 'redux-thunk';
 
 import { extraDependencies } from '../extraDependencies';
 
+interface MiddlewareAPI<D extends Dispatch = Dispatch<AnyAction>, S = any> {
+    dispatch: D;
+    getState(): S;
+}
+
+interface Middleware<_DispatchExt = {}, S = any, D extends Dispatch = Dispatch<any>> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)